src/App.tsx

```diff
@@ -1,61 +1,43 @@
+import { useState } from 'react';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { NewTodo } from './components/NewTodo';
+import { TodoList } from './components/TodoList';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
 
 export const App = () => {
+  const [listTodo, setListTodo] = useState([...todosFromServer]);
```

Comment:
It's not secure to directly use imported data in state. Consider copying data to the local variable first and then use it in the state.